defining the "network element" inline to provide the definition ahead of using it, uses throughput advice instead of bitrate (#114)
mjoras/SCONE-PROTOCL: documents/charter.md

bitrate termed "throughput advice" — to the endpoint originating the UDP
5-tuple. 

> This mechanism will allow an application to receive notifications from network elements, a "network element" is defined as anything on 
> the path of a UDP 5-tuple that is capable of dropping or delaying packets, containing throughput
> advice for both upstream and downstream traffic.

Reviewer: ```suggestion
This mechanism will allow an application to receive notifications containing throughput
advice for both upstream and downstream traffic from any network elements capable of dropping or delaying packets on 
the path of a UDP 5-tuple.
```

Reviewer: The text I'm commenting on seems to insert a complete sentence in the middle of another complete sentence. Does this suggestion seem clear?

Author: the proposal is much improved. thanks.
